authzed/api/v2/bulk_service.proto

```diff
@@ -0,0 +1,46 @@
+syntax = "proto3";
+package authzed.api.v2;
+
+option go_package = "github.com/authzed/authzed-go/proto/authzed/api/v2";
+option java_package = "com.authzed.api.v2";
+
+import "google/api/annotations.proto";
+import "validate/validate.proto";
+
+import "authzed/api/v2/core.proto";
+
+// BulkService exposes a number of APIs that can be used for managing
+// relationships in bulk.
+service BulkService {
+
+  // BulkLoadRelationships is a faster path to writing relationships many at
```

Comment:
BulkLoadRelationships is a faster path to writing a large number of relationships at once